tests/Aspire.Playground.Tests/.runsettings

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Do we need this copy of this file? Can we use the default one in the root?

Reply:
This is to allow the source being buildable from outside the repo, as we wouldn't have `eng/testing/.runsettings` available there.

The alternative would be to copy it for the archive, and also have a `<RunSettingsFilePath Condition="'$(RepoRoot)' == ''">$(MSBuildThisFileDirectory).runsettings</RunSettingsFilePath>`.

Reply:
Can we just have the root `.runsettings` file checked into the repo, and then the infrastructure copies it to the correct place in the helix payload?

Reply:
This will get covered by the follow up task of building on the build machine, and only running on helix.